src/__tests__/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../components/App";
+
+describe("App", () => {
+  const { asFragment } = render(<App />);
+
+  test("renders correctly", () => {
+    expect(asFragment()).toMatchSnapshot();
+  });
```

Comment:
Typical layout for this would be:
```
  test("renders correctly", () => {
    const { asFragment } = render(<App />);

    expect(asFragment()).toMatchSnapshot();
  });
```
Since there is only one test happening in this file its not super critical, typically the convention is to render an instance of whatever you are testing inside each test block. Also there should be a clear line between the setup and the block of assertions for each test. This is just as a general point of convention to make tests easier to read at first glance